src/rpc/blockchain.cpp

```diff
@@ -2729,6 +2814,7 @@ void RegisterBlockchainRPCCommands(CRPCTable& t)
         {"hidden", &waitforblockheight},
         {"hidden", &syncwithvalidationinterfacequeue},
         {"hidden", &dumptxoutset},
+        {"hidden", &loadtxoutset},
```

Comment:
In commit "rpc: add loadtxoutset" (82a8744f1384f07dc30aaa6339b0b8fb32dc3af0)

Should dumptxoutset and loadtxoutset still be hidden?